test/widgets/clipboard_test.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:checks/checks.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:zulip/model/binding.dart';
+import 'package:zulip/widgets/clipboard.dart';
+
+import '../flutter_checks.dart';
+import '../model/binding.dart';
+import '../test_clipboard.dart';
+
+void main() {
+  setUp(() async {
+    TestZulipBinding.ensureInitialized();
+    TestWidgetsFlutterBinding.ensureInitialized();
+    TestDefaultBinaryMessengerBinding.instance.defaultBinaryMessenger.setMockMethodCallHandler(
+      SystemChannels.platform,
+      MockClipboard().handleMethodCall,
```

Comment:
This one is good to do in the per-test setup, though, because it resets the clipboard's data.